helm-maintainers-onboarding-guide.md

```diff
@@ -107,7 +107,7 @@ Core maintainers take turns triaging the issue queue. The responsibilities of a
 - Features:
   - If it violates the SemVer rules, open a dialog on whether there is an alternate way to do this that will not break our SemVer rules
   - If it is a major feature (new subcommand, new way of doing something), ask if it can be implemented as a plugin or extension
-  - If a major change is warranted, suggest opening a [HIP](https://github.com/helm/community/blob/main/hips/hip-0001.md)
+  - All features and breaking changes MUST be described in a [HIP](https://github.com/helm/community/blob/main/hips/hip-0001.md), and that HIP must pass the usual process. (See HIP 12: "[Helm 4 Development Process](https://github.com/helm/community/blob/main/hips/hip-0012.md".)
```

Comment:
The double quotes and period at the end of this sentence  are misplaced, causing the markdown to render improperly. Should be

```
(See HIP 12: "[Helm 4 Development Process](https://github.com/helm/community/blob/main/hips/hip-0012.md)").
```